src/js/wrapper.ts

```diff
@@ -105,7 +105,7 @@ export const NATIVE: SentryNativeWrapper = {
         this is a signal that the app would crash and android would lose the breadcrumbs by the time the app is restarted to read
         the envelope.
       */
-      if (event.exception?.values?.[0]?.mechanism?.handled) {
+      if (event.exception?.values?.[0]?.mechanism?.handled != false && event.breadcrumbs) {
```

Comment:
`m`: I think the comment above needs some update. Otherwise, it will be confusing.

Reply:
Good catch